UserInterfaces/LaunchViewController.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LaunchViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/02/26.
+//
+
+import Foundation
+
+import UIKit
+
+class LaunchViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        sleep(2)
```

Comment:
is there a reason that we are making the app wait here?

Reply:
I thought it would be better to delay the launch screen so that the users can read the writing but I have removed it for now.